.circleci/config.yml

```diff
@@ -459,15 +459,13 @@ jobs:
           command: |
             #
             # This block should not cause a test failure and block PRs.
-            # The shell should never error and exit 0 to indicate success.
             #
             set +o errexit
-            npm run build-css                                      \
-              && npm run preanalyze-bundles                        \
-              && node bin/icfy-analyze.js                          \
-              && mv client/stats.json client/chart.json client/style.json "$CIRCLE_ARTIFACTS/icfy" \
+            npm run build-css                                                                       \
+              && npm run preanalyze-bundles                                                         \
+              && node --max-old-space-size=4096 bin/icfy-analyze.js                                 \
```

Comment:
Seems like a good size, these are the machine limits:

![Screen Shot 2020-02-12 at 12 04 05](https://user-images.githubusercontent.com/841763/74329290-cf315400-4d8f-11ea-987f-62fac30a53e1.png)

Might be in danger of passing machine limits, but we can let them fail if that happens.